Source/WebCore/PAL/pal/PALSwift.h

```diff
@@ -28,8 +28,39 @@
 #include <cstdint>
 #include <wtf/Vector.h>
 
+namespace Cpp {
+
 using VectorUInt8 = WTF::Vector<uint8_t>;
 using SpanConstUInt8 = std::span<const uint8_t>;
+using OptionalVectorUInt8 = std::optional<WTF::Vector<uint8_t>>;
+
+
+// FIXME: remove when swift support is available rdar://118026392
+inline OptionalVectorUInt8 toOptionalVectorUInt8(VectorUInt8 val)
+{
+    return val;
+}
+
+inline VectorUInt8 createVectorUInt8FromSpan(SpanConstUInt8 span)
+{
+    return VectorUInt8(span);
+}
+
+inline void appendSpanToVectorUInt8(const SpanConstUInt8& span, VectorUInt8& vec)
+{
+    vec.append(span);
+}
+
+namespace Unsafe {
+
+inline SpanConstUInt8 createConstUInt8Span(const uint8_t * start, size_t length)
+{
+    return { start, length };
+}
```

Comment:
This abstraction doesn't meet our unsafe design goal because it returns the unsafe pointer in a span rather than returning a safe abstraction.

Reply:
I have left it inside Unsafe intentionally for that reason. the other way would be to never create a span from withUnsafeBytes and just always create a Vector by Copying elements in a loop.
Span constructors are not available in swift as they are templated.